### ISSUE TITLE
Is another .NET semi-official allowed?

### ISSUE THREAD

**Issue by ttutisani** [cucumber/docs#449]
I have a .NET based framework that uses [gherkin](https://github.com/cucumber/cucumber/tree/master/gherkin) for .NET to parse feature files and execute scenarios via Xunit. Here it is: https://github.com/ttutisani/Xunit.Gherkin.Quick
It would qualify as semi-official since it's based on gherkin by Cucumber but is hosted not under Cucumber.

I was thinking if I could add the link to my framework on this page: https://cucumber.io/docs/installation/

I am asking this question because I don't see any other platform having more than one option, so I am wondering what would be the route: do I add it to the existing .net installation page (https://cucumber.io/docs/installation/dotnet/) or create a new one?

Thank you!

**Comment by aslakhellesoy** [comment]
Hi @ttutisani - that's great to see! Feel free to send a PR against the `cucumber/docs.cucumber.io` repo to add it.

**Comment by ttutisani** [comment]
Will do. Thanks!

**Comment by ttutisani** [comment]
All right, I tried, and I got stuck. Who can help, please?

I followed the instructions on the home page of the repo to pull down the image and run it as a container: https://github.com/cucumber/docs.cucumber.io

I pulled down the code.
```bash
git clone https://github.com/cucumber/docs.cucumber.io.git
cd docs.cucumber.io
```

Next, I ran the command to start the container:
```bash
./docker_shell.sh
```

Images pulled down fine to recreate the necessary `cucumber/hugo` image, However, when it tries to start the container, it gets the error:
```
Error: Unable to locate Config file. Perhaps you need to create a new site.
       Run `hugo help new` for details. (Config File "config" Not Found in "[/app]")
```

I am definitely missing something which is not mentioned in the document, or I may be looking at the wrong document. Please direct me to the right place to fix this issue.

Thank you!

**Comment by olleolleolle** [comment]
As a workaround, perhaps you can use the "edit pen on GitHub" workflow to add your markup?

**Comment by ttutisani** [comment]
That is a good idea. I will try that too.
Thanks!

**Comment by ttutisani** [comment]
I just opened a pr #450 and the preview looks just right. I hope it will get merged soon. Thanks for your help!

**Comment by mlvandijk** [comment]
The PR has merge conflicts. Could you resolve them please @ttutisani ?

**Comment by ttutisani** [comment]
@mlvandijk thanks for the response! The PR is fixed and has no merge conflicts now. Please merge if no other concerns.

**Comment by mlvandijk** [comment]
Closed by #450
